app/src/main/res/layout/content_main.xml

```diff
@@ -4,30 +4,162 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@drawable/hill_background"
     app:layout_behavior="@string/appbar_scrolling_view_behavior"
     tools:context="io.neurolab.MainActivity"
     tools:showIn="@layout/app_bar_main">
 
     <RelativeLayout
+        android:id="@+id/parent_relative_layout"
         android:layout_width="match_parent"
         android:layout_height="match_parent">
 
-        <ImageView
```

Comment:
The animation is something which has to showcased at the forefront right now. @mariobehling can tell you more about this. Right now I can see that you are REMOVING the animation. @suresh-gandhi please take this to another fragment/activity. Otherwise the PR seems fine. Cheers!

Reply:
@jddeep Thanks for letting me know. I am waiting for the approval of our mentors @CloudyPadmal @mariobehling. If they suggest the same I would surely incorporate your changes.